apps/api/src/app/events/usecases/parse-event-request/parse-event-request.command.ts

```diff
@@ -37,6 +37,10 @@ export class ParseEventRequestBaseCommand extends EnvironmentWithUserCommand {
   @IsOptional()
   @IsEnum(TriggerRequestCategoryEnum)
   requestCategory?: TriggerRequestCategoryEnum;
+
+  @IsOptional()
+  @IsNumber()
+  priority?: number;
```

Comment:
This change will result in the `priority` property becoming available on the OpenAPI spec at https://api.novu.co/openapi.yaml, which in turn, is used to generate documentation at https://docs.novu.co/api-reference/overview. 

Given that this property is only usable in self-hosted via `IS_DOCKER_HOSTED === 'true'` in the implementation, we also need to conditionally apply [@ApiHideProperty](https://docs.nestjs.com/openapi/cli-plugin#:~:text=ApiProperty%20unless%20%40-,ApiHideProperty,-is%20used) when the same condition is met, to ensure the property doesn't become available in public documentation, which pertains only to usage of Novu Cloud at present.